docs/source/includes/gcloud_setup.rst

```diff
@@ -1 +1,24 @@
 Follow the Windows, Mac OS X or Linux instructions to install gcloud on your local machine: https://cloud.google.com/sdk/
+
```

Comment:
Here for the gcloud install and for the genomics tools install, it is handy to have this content inline so that users don't need to navigate elsewhere, but it also means that its another copy of instructions to keep in sync.

@mbookman what do you think?